server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
+    doc: 'The url of the Firefox Sync tokenserver',
+    env: 'SYNC_TOKENSERVER_URL',
```

Comment:
> Conversely, doing it in the server makes it more likely to accidentally change due to e.g. a configuration problem.

Outside of the scope of this PR, but this indicates to me we need a more robust, easier to update, configuration management process. If we see a configuration problem, we want to update in a few minutes, not wait for a new stack to be built and deployed. Of course, we can by ssh'ing to the various boxes, updating config, and restarting, but that's neither error-resistant nor scalable.

> Perhaps we need to replace it with something significantly more light-weight, that just runs e.g. once an hour and checks whether it can successfully connect to sync, 

How often is Sync data sync'd, would we know right away if there was an uptick in the number of Sync connection errors? Can notifications occur w/o new browser code to force check once per hour?

If we do encounter a connection error because of a misconfigured URL or the Sync token server just being down, what does the browser do to recover?